utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
+prjFileName="fsdaToolboxPackaging.prj";
 toolboxOption=matlab.addons.toolbox.ToolboxOptions(prjFileName);
```

Comment:
Follow MathWorks standard for customer facing code, there should be space on both the sides of "="

Reply:
Added space before and after =